ccd-definition/CaseField/CaseField-COS-nonprod.json

```diff
@@ -1,4 +1,53 @@
 [
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "cosNotifyClaimDetails1",
+    "Label": " ",
+    "FieldType": "NotifyClaimDetailsCoS",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "cosNotifyClaimDetails2",
+    "Label": " ",
+    "FieldType": "NotifyClaimDetailsCoS",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "cosLabelForClaimant1",
+    "Label": "Claimant's name - ${applicant1.partyName}",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "cosLabelForClaimant1_1",
+    "Label": "Claimant's name <br /> ${applicant1.partyName}",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "cosLabelForClaimant2_1",
+    "Label": "Claimant's name <br /> ${applicant1.partyName}",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
```

Comment:
I would suggest using the variable I added "${respondent1.partyName}" as we shouldnot be depending on variables defined specific for other tab not related to CoS.

Reply:
${respondent1.partyName} and other respondent1 values are not loaded at CoS notify detail page so have to use this value

Reply:
fixed